### DISCUSSION TITLE
更新到最新dev之后,meta设置消失

### DISCUSSION THREAD

**Discussion by MoePotatoChan** [vernesong/OpenClash#4085] [Q&A]
最近更新了最新的dev，发现ip嗅探这些全部不见了，在哪可以打开呢，求助
版本
![image](https://github.com/user-attachments/assets/79545425-a406-430e-b83a-56b5a20a482b)
是arm64的机器
设置如图
![image](https://github.com/user-attachments/assets/9707b9c5-fcf6-4ea4-a1ca-e58d3562910b)

谢谢谢

**Comment by refined-fish** [comment]
最新版本没有dev和tun内核了，只有meta内核，直接到覆写设置中修改meta选项即可，你也可以直接写进配置文件中
另外你截的图不是覆写设置，覆写设置页面在第二页

**Reply by omega0503** [reply]
我升级meta内核之后，怎么都没法用了🙁
